lib/getstats.js

```diff
@@ -316,6 +323,38 @@ function chromeGetTrackStats(peerConnection, track) {
   });
 }
 
+/**
+ * Get the standardized statistics for a particular MediaStreamTrack in Edge.
+ * @param {RTCPeerConnection} peerConnection
+ * @param {MediaStreamTrack} track
+ * @returns {Promise.<StandardizedTrackStatsReport>}
+ */
+function edgeGetTrackStats(peerConnection, track) {
+  return waitForEdgeIceCompletion(peerConnection).then(function() {
+    return peerConnection.getStats(track);
+  }).then(function(response) {
+    return standardizeEdgeAndFirefoxStats(response);
+  });
+}
+
+function waitForEdgeIceCompletion(peerConnection) {
+  const iceConnectionStatesToWait = new Set(['checking', 'connected', 'new']);
+  if (!iceConnectionStatesToWait.has(peerConnection.iceConnectionState)) {
+    return Promise.resolve();
+  }
+  return new Promise(function(resolve, reject) {
+    const timeoutToFail = setTimeout(reject, 5000, new Error('failed'));
```

Comment:
I'm not sure if we should include this. Would an unresolved Promise be better than a Promise that rejects arbitrarily after 5s?